databases/gorm/gormdb.go

```diff
@@ -13,29 +14,60 @@ type gormdb struct {
 	db *gorm.DB
 }
 
-func Connect(dsn string) *gormdb {
+func NewGormDB(db *gorm.DB) *gormdb {
+	return &gormdb{
+		db: db,
+	}
+}
```

Comment:
Connect was doing the same thing, seperating these might be a good choice, but leave a todo when that's too far from pull request goal.